comfy/latent_formats.py

```diff
@@ -95,6 +95,7 @@ def __init__(self):
             [ 0.0542,  0.1545,  0.1325],
             [-0.0352, -0.1672, -0.2541]
         ]
+    taesd_decoder_name = "previewer.safetensors"
```

Comment:
How about sdcascade_previewer.safetensors or any other name that tells the user which model it is.

Reply:
I changed it to cascade_previewer for now